### PR TITLE
Create music album class

### PR DESCRIPTION
**Pull request by Estherstarr**
In this PR, I implemented the following
- [ ] Create music album class

### CODE REVIEW

File: classes/music_album.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require_relative './item'
```
Comment (RWUBAKWANAYO) [inline classes/music_album.rb:1]: - [ ] Pllease change this route from its origin
```suggestion
require_relative '../item'
```

File: classes/music_album.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+class MusicAlbum < Item
+  attr_reader :album_name, :artist_name, :publish_date, :on_spotify
+
+  def initialize(album_name, artist_name, publish_date, on_spotify)
+    super(publish_date)
```
Comment (RWUBAKWANAYO) [inline classes/music_album.rb:7]: - [ ] Archived property accessed from item class
```suggestion
    super(publish_date, archived: archived)
```

**Review by RWUBAKWANAYO — CHANGES_REQUESTED** [review]
Hi, @Estherstarr 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following: 
- Nicely structured documentation.✔️
- Clean Ruby code and style guide.✔️
- No linter errors.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏
***
- [ ] to get timeconvertion you can use `Time.now` instead of `Time.year`
- [ ] genre, author, source, label can be defined outside of `item constructor` as they requested in instructions
https://github.com/kamzzy/ruby-group-project/blob/7df227c2f21b8e0fc87c6ab3ebce29e18c718be6/item.rb#L2

File: classes/music_album.rb (hunk 2 above)
Comment (Estherstarr) [reply classes/music_album.rb]: @RWUBAKWANAYO 
The required changes has been made

**Review by RWUBAKWANAYO — APPROVED** [review]

**Pull request merged**